openmrs-client/versions.gradle

```
@@ -0,0 +1,51 @@
+ext.versions = [:]
+
+versions.compileSdk = 29
+versions.minSdk = 16
+versions.targetSdk = 29
+versions.appVersion = "2.8.3"
+
+//Libraries
```

```suggestion
// Libraries
```
Just to follow openmrs conventions.